apps/examples/testcase/le_tc/kernel/tc_timer.c

```diff
@@ -122,11 +193,48 @@ static void tc_timer_timer_set_get_time(void)
 	st_timer_spec_set.it_value.tv_sec = 1;
 	st_timer_spec_set.it_value.tv_nsec = 0;	/* expire; */
 
+	/* Null timer ID check for timer_settime */
+
+	ret_chk = timer_settime(NULL, 0, &st_timer_spec_set, NULL);
+	TC_ASSERT_EQ_CLEANUP("timer_settime", ret_chk, ERROR, timer_delete(timer_id));
+	TC_ASSERT_EQ_CLEANUP("timer_settime", errno, EINVAL, timer_delete(timer_id));
+
+	/* Null value check for timer_settime */
+
+	ret_chk = timer_settime(timer_id, 0, 0, NULL);
```

Comment:
3rd param type is struct itimerspec *.
you'd better use NULL instead of 0

Reply:
done